admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/controller/BiometricTypeControllerTest.java

```diff
@@ -77,7 +77,7 @@ public void t9createBiometricTypeFailTest() throws Exception {
 						+ "    \"isActive\": true,\n" + "    \"langCode\": \"eng\",\n" + "    \"name\": \"Iris\"\n"
 						+ "  }\n" + "}"))
 
-				.andReturn(),"KER-MSD-105");
```

Comment:
why for failed test cases we are adding null or modify the test name

Reply:
Modified the name sir.